saleor/order/utils.py

```diff
@@ -274,13 +259,20 @@ def delete_order_line(line):
 def restock_order_lines(order):
     """Return ordered products to corresponding stocks."""
     country = get_order_country(order)
+    default_warehouse = Warehouse.objects.filter(
+        shipping_zones__countries__contains=country
+    ).first()
```

Comment:
Using the first warehouse from the list is our assumption for now - maybe it would be better to put that to some function? `get_default_warehouse`? It will be easier to find later when we decide to move it to a plugin or something.

Reply:
I think it not worth changing it because in the next PR we add warehouse to mutation input, and if we still need it we add this function to code.

Reply:
👍